analyses-ccg.Rmd

```diff
@@ -298,27 +298,35 @@ r_ccg <- counts_ccg_all %>%
   growth_rate() %>%
   rename("ccg" = "ccg_name")
 
+r_plot <- counts_ccg_all %>% fit_curve(model = "poisson") %>%
```

Comment:
These plots do not correspond to the calculated r-values (i.e. they use `keep_last(3*7)`)

Reply:
Does keep_last work here? I think I tried that before and I was struggling to get a plot.

Reply:
possibly not - it may need doing manually.  The key thing is the plots need to match with the number of days we've fitted the growth rate to.

Reply:
actually - it should work there.  Let me know if there's an issue and we can dig in to further